common/src/main/java/org/apache/drill/common/AutoCloseables.java

```diff
@@ -25,6 +25,11 @@
  */
 public class AutoCloseables {
 
+  public interface Closeable extends AutoCloseable {
```

Comment:
The change allows avoiding `catch` in try-with-resources when `close()` does not throw an exception.

Reply:
But java has already `Closeable` interface, that's confusing having custom one with the same name.

Reply:
It is on purpose:
- There is a minimal difference between Drill `Closeable` and Java 'Closeable/AutoCloseable`, so name reflects that.
- It won't be possible to use Drill `Closeable` in place of Java `Closeable/AutoCloseable` in case `close()` throws checked exception.
- Drill `Closeable` is not a top level interface, so where it is necessary to distinguish Drill `Closeable` from Java `Closeable` full name `AutoClosables.Closeable` should be used.